### ISSUE TITLE
atLeast() marks all invocations verified

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dlsgusrn7577/mockito#112]
```
This test fails:
-------------------------------------------------
public class SomeMethods {
  public void allowedMethod() {}
  public void disallowedMethod() {}
}

@Test(expected =
org.mockito.exceptions.verification.NoInteractionsWanted.class)
public void shouldFailBecauseDisallowedMethodWasCalled(){
  SomeMethods someMethods = mock(SomeMethods.class);

  someMethods.allowedMethod();
  someMethods.disallowedMethod();

  verify(someMethods, atLeast(1)).allowedMethod();
  verifyNoMoreInteractions(someMethods);
}
```

Original issue reported on code.google.com by `lukewpat...@gmail.com` on 14 Aug 2009 at 6:28
- Merged into: #116

**Comment by GoogleCodeExporter** [comment]
```
failing unit test patch and fix patch
```

Original comment by `lukewpat...@gmail.com` on 14 Aug 2009 at 6:30

Attachments:
- [mockito_issue_112_failing_unit_test.patch](https://storage.googleapis.com/google-code-attachments/mockito/issue-112/comment-1/mockito_issue_112_failing_unit_test.patch)
- [mockito_issue_112_fix.patch](https://storage.googleapis.com/google-code-attachments/mockito/issue-112/comment-1/mockito_issue_112_fix.patch)

**Comment by GoogleCodeExporter** [comment]
```
ouch... embarassing bug :)

Thanks for patching. I'll apply it soon.
```

Original comment by `szcze...@gmail.com` on 17 Aug 2009 at 8:37

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision r1589.
```

Original comment by `szcze...@gmail.com` on 22 Aug 2009 at 9:11
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thanks again for reporting and patching! Good work.
```

Original comment by `szcze...@gmail.com` on 22 Aug 2009 at 9:12

**Comment by GoogleCodeExporter** [comment]
Original comment by `szcze...@gmail.com` on 11 Nov 2009 at 1:33
- Added labels: **Milestone-Release1.8.1**

**Comment by GoogleCodeExporter** [comment]
Original comment by `szcze...@gmail.com` on 11 Nov 2009 at 2:44
- Changed state: **Duplicate**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 852ae327ca.
```

Original comment by `szcze...@gmail.com` on 7 Oct 2010 at 5:21
- Changed state: **Fixed**
